Content.Server/Labels/Label/LabelSystem.cs

```diff
@@ -74,7 +75,7 @@ public void Label(EntityUid uid, string? text, MetaDataComponent? metadata = nul
 
             // Update label
             label.OriginalName ??= metadata.EntityName;
-            label.CurrentLabel = text;
+            label.CurrentLabel = FormattedMessage.EscapeText(text);
```

Comment:
This is not a correct fix. This should be sanitized when the value is output to the examine window instead.

Reply:
I did look into doing that. I opted not to do it, because I didn't want to cut off the potential for an item to have a colored name, or some other text effect in the examine window.

Reply:
> because I didn't want to cut off the potential for an item to have a colored name

Those already would not show up correctly.